design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -297,9 +305,10 @@ Note:
     </hardware>
     <joint name="joint1">
       <classType>ros2_control_components/PositionJoint</classType>
-      <stateInterfaceType>position</stateInterfaceType>
-      <param name="min_position_value">${-PI}</param>
-      <param name="max_position_value">${PI}</param>
+      <stateInterfaceType name="position">
```

Comment:
This would be this optional limits for the `stateInterfaceType`.

Nevertheless, I did error here, I think here we should have sensor and not joint component (tag in L306))